tutorial/stream-client.md

```diff
@@ -0,0 +1,261 @@
+```
+
+Create a `stream.html` file inside `stream-client/static` with the following code:
+
+```html
+<!doctype html>
+<html>
+  <head>
+    <title>Anbox Streaming SDK - Tutorial</title>
+    <script type="module" src="static/anbox-stream-sdk.js"></script>
+    <style>
+      html,
+      body,
+      #anbox-stream {
+        height: 100%;
+        margin: 0;
+        width: 100%;
+      }
+    </style>
+  </head>
+
+  <body>
+    <div id="anbox-stream"></div>
+    <script>
+      const sessionId = "{{ session_id }}";
+      class Connector {
+        async connect() {
+          var url = `${window.location.origin}/1.0/sessions/${sessionId}/join`;
+          const resp = await fetch(url, { method: "POST" });
+          if (resp === undefined || resp.status !== 200)
+            throw new Error("failed to join session");
+
+          const response = await resp.json();
+          if (response === undefined || response.status !== "success")
+            throw new Error(response.error);
+
+          return {
+            id: response.metadata.id,
+            websocket: response.metadata.url,
+            stunServers: response.metadata.stun_servers,
+          };
+        }
+
+        disconnect() {}
+      }
+      var stream;
+      window.onload = () => {
+        const connector = new Connector();
+        stream = new AnboxStream({
+          connector: connector,
+          targetElement: "anbox-stream",
+          enableStats: false,
+          stream: {
+            audio: true,
+            video: true,
+          },
+        });
+        stream.connect();
+      };
+    </script>
+  </body>
+</html>
+```
+
+Next, let's make a copy of the Anbox Stream Gateway server to allow the service to validate it:
+
+    # We use cat here to ensure gateway.crt has the users permissions
+    sudo cat /var/snap/anbox-cloud-appliance/common/gateway/server.crt > gateway.crt
+
+As a final step, download a copy of the Anbox streaming SDK and place it inside `stream-client/static`
+
+    curl -o stream-client/static/anbox-stream-sdk.js \
+        https://raw.githubusercontent.com/canonical/anbox-streaming-sdk/refs/heads/main/js/anbox-stream-sdk.js
+
+## Run the stream client
+
+To run the stream client, let's set up a virtual environment for Python to install necessary dependencies:
+
+    cd stream-client
+    python3 -m venv .venv
+    . .venv/bin/activate
+
+Now, we can use `pip` to install all required dependencies
+
+    pip3 install -r requirements.txt
+
+To run the stream client service application, we need to set the necessary environment variables and execute the service via flask.
+
+When installing and initialising the appliance, we would have configured the private IP address of the machine running the appliance. The gateway API is accessible on the local listen address of the appliance on port `9031/tcp`. So that's the IP address and port we use in the GATEWAY_URL.
+
+    export GATEWAY_URL=https://1.2.3.4:9031
+    export GATEWAY_API_TOKEN=xxx
```

Comment:
But what token should be input by the user here? Should we be explicit about how to do this? Or maybe link them to some other docs?

Reply:
We have that above under prerequisites but we should put to that in the description here

Reply:
I agree, I propose to chain the output of the command above here, so that it is used directly in the context where it is needed:
```suggestion
    export GATEWAY_API_TOKEN=$(anbox-cloud-appliance gateway account create your-username)
```

Reply:
We are going through a learning flow in the tutorial where we help the user prepare everything they need and then when they use it later, it helps them understand how various things work together.

So I have chosen to add text description about the token rather than chaining the commands